core/domain/email_domain.py

```diff
@@ -70,3 +70,37 @@ def thread_id(self):
         """
         return '.'.join(
             [self.entity_type, self.entity_id, self.id.split('.')[3]])
+
+
+class SendEmailInfo(python_utils.OBJECT):
```

Comment:
Regarding naming, this seems slightly misleading. Isn't this only being used for multi-recipient emails? Alternatively, you could update all email model interactions to use the domain structure passed in, then you could consolidate create() & create_multi().

Also, is this actually used for sending an email or just recording that an email has been sent? The name & doc string should make this clear.